eeg_matchingpennies/sub-05/eeg/sub-05_task-matchingpennies_events.tsv

```diff
@@ -1,301 +1,901 @@
-1698	0	left	33.3333333333	right_hand.png	1	8491779	3	71	left	1	13	-25.0
-1703	0	left	116.666666667	right_hand.png	1	8515068	3	72	left	1	13	1.8
-1707	0	right	66.6666666667	left_hand.png	2	8538065	3	73	right	1	13	10.4
-1712	0	left	83.3333333333	left_hand.png	1	8561166	3	74	right	1	13	-1.8
-1716	0	left	50.0	right_hand.png	1	8584099	3	75	left	1	13	-6.0
-1723	0	left	183.33333333299998	right_hand.png	1	8619043	3	76	left	1	14	7.8
-1730	0	left	100.0	right_hand.png	1	8650484	3	77	left	1	15	-22.2
-1734	0	right	33.3333333333	left_hand.png	2	8673341	3	78	right	1	15	-11.2
-1739	0	left	50.0	right_hand.png	1	8696268	3	79	left	1	15	-24.0
-1743	0	right	66.6666666667	right_hand.png	2	8719308	3	80	left	1	15	-8.8
-1752	0	left	116.666666667	right_hand.png	1	8760067	3	81	left	1	16	-10.0
-1756	0	right	133.33333333299998	right_hand.png	2	8783419	3	82	left	1	16	-10.8
-1761	0	left	83.3333333333	left_hand.png	1	8806518	3	83	right	1	16	-21.0
-1765	0	left	33.3333333333	left_hand.png	1	8829370	3	84	right	1	16	-10.6
-1770	0	left	133.33333333299998	left_hand.png	1	8853139	3	85	right	1	16	-2.8
-1778	0	left	133.33333333299998	right_hand.png	1	8894086	3	86	left	1	17	-0.2
-1783	0	right	83.3333333333	left_hand.png	2	8917187	3	87	right	1	17	-12.4
-1788	0	right	33.3333333333	right_hand.png	2	8940038	3	88	left	1	17	-11.4
-1792	0	left	166.66666666700002	right_hand.png	1	8963556	3	89	left	1	17	-18.0
-1797	0	left	116.666666667	right_hand.png	1	8986845	3	90	left	1	17	-17.6
-1809	0	left	116.666666667	left_hand.png	1	9049561	3	91	right	1	19	0.8
-1814	0	left	100.0	left_hand.png	1	9072745	3	92	right	1	19	-20.0
-1819	0	left	83.3333333333	right_hand.png	1	9095824	3	93	left	1	19	-2.2
-1823	0	right	50.0	right_hand.png	2	9118763	3	94	left	1	19	-12.0
-1828	0	left	50.0	right_hand.png	1	9141696	3	95	left	1	19	-11.8
-1832	0	right	50.0	right_hand.png	2	9164627	3	96	left	1	19	-15.6
-1837	0	right	83.3333333333	left_hand.png	2	9187727	3	97	right	1	19	-27.6
-1842	0	left	100.0	right_hand.png	1	9210913	3	98	left	1	19	-5.6
-1846	0	right	116.666666667	left_hand.png	2	9234202	3	99	right	1	19	-29.8
-1851	0	left	116.666666667	right_hand.png	1	9257470	3	100	left	1	19	-24.2
+onset	duration	trial_type	response_time	stim_file	value	sample	stage	trial	bci_prediction	bci_prediction_valid	n_repeated	latency
+15.038933333332999	n/a	right	116.666666667	left_hand.png	n/a	n/a	1	1	right	1	2	-4.2
+18.038933333333	n/a	right	116.666666667	left_hand.png	n/a	n/a	1	1	right	1	2	-4.2
+18.1556	0	right	116.666666667	left_hand.png	2	90778	1	1	right	1	2	-4.2
+19.8922	n/a	right	100.0	right_hand.png	n/a	n/a	1	2	left	1	2	-2.4
+22.8922	n/a	right	100.0	right_hand.png	n/a	n/a	1	2	left	1	2	-2.4
+22.9922	0	right	100.0	right_hand.png	2	114961	1	2	left	1	2	-2.4
+24.5304666666667	n/a	left	33.3333333333	right_hand.png	n/a	n/a	1	3	left	1	2	-17.2
+27.5304666666667	n/a	left	33.3333333333	right_hand.png	n/a	n/a	1	3	left	1	2	-17.2
+27.5638	0	left	33.3333333333	right_hand.png	1	137819	1	3	left	1	2	-17.2
```

Comment:
look at the three rows above --> they show a pattern that repeats throughout the events tsv files:
- countdown onset
- countdown offset
- button release

all belong to a single trial ... but have different `sample` values ... we only know the sample for "button release". etc.